### PR TITLE
Set SDK version to v0.47x.0

### LINKED ISSUE #1148: Upgrade to cosmos-sdk proto 0.47.x
Bump [cosmos-sdk proto dependency](https://github.com/CosmWasm/wasmd/blob/558147d054c88840a804a3ec4e0bba5b3cb5905a/proto/buf.yaml#L10) to 0.47.x commit when available [here](https://github.com/cosmos/cosmos-sdk/blob/main/proto/README.md)

### PR DESCRIPTION
**Pull request by alpe**
Resolves #1148

🎉 SDK [v0.47.0](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.47.0)

### CODE REVIEW

File: proto/buf.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: v1
 name: buf.build/cosmwasm/wasmd
 deps:
   - buf.build/cosmos/cosmos-proto
-  - buf.build/cosmos/cosmos-sdk:2aa7ff2b23df473a85b7a7fe1884105d #v0.47.0-rc
+  - buf.build/cosmos/cosmos-sdk:8c515ebc07ee4514aabcaf3a584a1d1a #v0.47.0
```
Comment (julienrbrt) [inline proto/buf.yaml:5]:
```suggestion
  - buf.build/cosmos/cosmos-sdk:v0.47.0
```

Comment (alpe) [reply proto/buf.yaml]: This is nice! Thanks for letting me know

File: INTEGRATION.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ hardware it runs on.
 
 | wasmd | Cosmos SDK |
 |:-----:|:----------:|
+| v0.40 |  v0.47.0   |
```
Comment (pinosu) [inline INTEGRATION.md:22]: 👍

File: go.mod [hunk 3]
```diff
@@ -5,15 +5,15 @@ go 1.19
 require (
 	github.com/CosmWasm/wasmvm v1.2.1
 	github.com/cosmos/cosmos-proto v1.0.0-beta.2
-	github.com/cosmos/cosmos-sdk v0.47.0-rc3.0.20230308212038-818f6a047eeb
+	github.com/cosmos/cosmos-sdk v0.47.0
```
Comment (pinosu) [inline go.mod:8]: 👍

**Pull request merged**